app/assets/javascripts/components/todo_list_container.js.jsx

```diff
@@ -7,19 +7,35 @@ Chk.TodoListContainer = (function () {
       todos: React.PropTypes.array.isRequired
     },
 
+    filterList: function(event){
+      if (event.target.checked){
+        var updatedList= this.props.todos 
+        _.filter(updatedList, function(todo){return todo.complete_date})
```

Comment:
Hm my commit didnt go through.  This was supposed to be: `updatedList = _.reject(updatedList, function(todo){return todo.completed_date;})`